File: 2024/09 - September/20. Facing the Sun/Solution.py
class Solution:
    def countBuildings(self, height):
        if not height:
            return 0
        
        res, tallest = 1, height[0]
        for i in range(1, len(height)):
            if height[i] > tallest:
                res += 1
                tallest = height[i]
        
        return res

File: 2024/09 - September/20. Facing the Sun/Solution.java
class Solution 
{
    public int countBuildings(int[] height) 
    {
        if(height.length == 0)
            return 0;
            
        int res = 1, tallest = height[0];
        for(int i=1; i<height.length; i++)
        {
            if(height[i] > tallest)
            {
                res++;
                tallest = height[i];
            }
        }
        
        return res;
    }
}
